README.md

```diff
@@ -75,6 +75,11 @@ height = 1440
 outputFile = ~/Videos/ReplaySorcery/%F_%H-%M-%S.mp4
 ```
 
+Note that if the geometry is not correctly defined (using `offsetX`, `offsetY`, `width` and `height`) and not in line with the actual layout of your monitors, the following warning may start appearing in the log output and potentially increase memory consumption over time [#10](https://github.com/matanui159/ReplaySorcery/issues/10):
```

Comment:
Does it increase memory usage more than it does with correct parameters? If it does that's probably an issue cause it shouldn't (it should just generate garbage frames).

Reply:
I would say so yes. Before I correctly configured the geometry of the recording, the memory consumption was climbing steadily and never quite settling. After I realized what was happening (due to the error messages and your help) and fixed the issue (by making sure my second monitor was activated on boot), it seems somewhat more stable in terms of the amount of memory allocated.

Perhaps it's not related at all, but getting rid of those errors was the only thing that I explicitly remember doing...

Reply:
In any case, feel free to leave out the bit regarding the increase in memory consumption, if you are not quite convinced :)

Reply:
I'm busy all day today but tomorrow I'll look into why that is happening

Reply:
I think that width that is not divisible by 4 might be slower to process since it is no longer dword aligned.

Reply:
Possibly but would be too pedantic to kick people out and could be worked around by adding padding to each row

Reply:
benchmarks should be done. If it's slow enough then I think the config parser could just round up the resolution to make it divisible by 4 (and also by 2 to avoid these X warnings). Any normal resolution is divisible by 4 already anyway. It's only if you select your own small rectangle that you get into this problem.

Reply:
![image](https://user-images.githubusercontent.com/5038541/88621179-1e233980-d0e3-11ea-9a6d-917f697b1997.png)

Also you never know for sure what kind of setups people have out there. What if they have an offset and rounding up makes it go over the edge of the display? If the bottleneck really comes down to word sizes we can just add padding to rows, I made sure to keep stride independent from width when representing frames. At the end of the day however I doubt theres any benefit to wasting time on such a thing, unless someone comes along and specifically has this problem that can be exactly proven to be caused by this one very tiny (relative speaking) thing.

Also these X warnings are not due to it not being divisible, its to do with it being out of bounds.

+ frames almost always contain 4 bytes per pixel anyway.

Reply:
I tested around with this a bit and I could not replicate the issue of climbing memory usage when the warnings are generated. I did however add a check during init to make sure the rectangle does fit within the display.